Update CHANGELOG.md for #4274 (#4308)
Unity-Technologies/ml-agents: com.unity.ml-agents/CHANGELOG.md

calls to `StatsRecorder.Add()` with the same key in the same step will no
longer overwrite each other. (#4236)
#### ml-agents / ml-agents-envs / gym-unity (Python)

> - The supported versions of `numpy` ml-agents-envs was changed to not allow 1.19.0 or later. This was done to reflect

Reviewer: ```suggestion
- The versions of `numpy` supported by ml-agents-envs was changed to disallow 1.19.0 or later. This was done to reflect
```

Author: That's better, but I think it needs to be "versions ... _were_ changed". Will fix along with the merge conflicts.
